config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/IncludeAssertionBuilder.java

```diff
@@ -0,0 +1,84 @@
+package com.ca.apim.gateway.cagatewayconfig.bundle.builder;
+
+import com.ca.apim.gateway.cagatewayconfig.beans.Bundle;
+import com.ca.apim.gateway.cagatewayconfig.beans.DependentBundle;
+import com.ca.apim.gateway.cagatewayconfig.beans.MissingGatewayEntity;
+import com.ca.apim.gateway.cagatewayconfig.beans.Policy;
+import com.ca.apim.gateway.cagatewayconfig.util.paths.PathUtils;
+import com.ca.apim.gateway.cagatewayconfig.util.policy.PolicyXMLElements;
+import com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentParseException;
+import org.w3c.dom.Element;
+
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static com.ca.apim.gateway.cagatewayconfig.util.policy.PolicyXMLElements.POLICY_GUID;
+import static com.ca.apim.gateway.cagatewayconfig.util.policy.PolicyXMLElements.STRING_VALUE;
+import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.getSingleElement;
+
+public class IncludeAssertionBuilder implements PolicyAssertionBuilder {
```

Comment:
don't we need to apply unique naming strategy for included policy name?

Reply:
included policy is referred by id so we need not rename the original policy name

Reply:
referred by id in the policy.xml?

Reply:
Yes,

&lt;L7p:Include&gt;&#13;
            &lt;L7p:PolicyGuid stringValue="891fda14-0fc0-4e84-82fb-359c5aa14cf8"/&gt;&#13;
        &lt;/L7p:Include&gt;&#13;

Reply:
strange, we are simplifying the IncludeAssertion to remove the ID while exporting.